### ISSUE TITLE
Access Point is not visible even though I get AP Enable sucks message

### ISSUE THREAD

**Issue by sharmmoh1983** [sabhiram/raspberry-wifi-conf#29]

**Comment by mike-pun** [comment]
I am having the same problem... anyone got this working lately?

**Comment by hakankaraduman** [comment]
Same problem for me

**Comment by sabhiram** [comment]
Are you guys on Pi3s?

**Comment by mike-pun** [comment]
Yes we are (my group)

**Comment by sabhiram** [comment]
If you are able to debug this - I would be interested to know if re-starting the application resolves this and shows the AP. I saw this once on my PI3 and might be some kind of glitch.  If I have a few more data points, it will be easier to debug.

**Comment by chingling123** [comment]
same problem here.... if I reboot the wifi not appears but if I run the code the ap wifi works.. :(

**Comment by sabhiram** [comment]
oh very interesting - perhaps this is an issue with the startup script. 

How are you guys enabling the autostart?

**Comment by chingling123** [comment]
I tried only in rc.local and via services... the same problem. The ESSID in iwconfig coming with off/any

**Comment by sabhiram** [comment]
But when this is run from the command line - you confirm that it behaves as expected? Let me try reproducing that first thing tomorrow. Thanks for the reply!

**Comment by chingling123** [comment]
Yes, if I run in command line it's ok. Another thing that I found is in you regex lines you are trying to find inet-addr and HWaddr and in my ifconfig it's return inet and ether I have changed the regex to work and when you check the ip before turns in ap mode...

**Comment by sabhiram** [comment]
This might be part of another issue. I will need to think about how to handle different ways in which the inet addr is being reported. Perhaps there is a slicker way to access the required data.

**Comment by RuneSamyn** [comment]
I had the same issue and I solved it with these 2 lines in the beginning:
- sudo systemctl unmask hostapd
- sudo systemctl enable hostapd

https://www.raspberrypi.org/documentation/configuration/wireless/access-point-routed.md
